### PR TITLE
MAGECLOUD-5139: Add possibility to edit index.php in production mode

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

Adds instruction on how to mount custom `index.php` file on Cloud Docker

- https://magento2.atlassian.net/browse/MAGECLOUD-5139

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-containers-service.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-containers-service.md [hunk 1]
```diff
@@ -170,13 +170,26 @@ To mount custom NGINX configuration file using volumes:
...
+   ```yaml
+   services:
+     web:
+      volumes:
+        - ./.docker/nginx/etc/nginx.conf:/etc/nginx/nginx.conf
+        - ./.docker/nginx/etc/vhost.conf:/etc/nginx/conf.d/default.conf
    ```
 
+{:.procedure}
+To mount custom index.php file using volumes:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md]:
```suggestion
To mount the custom index.php file using volumes:
```

File: src/cloud/docker/docker-containers-service.md [hunk 2]
```diff
@@ -170,13 +170,26 @@ To mount custom NGINX configuration file using volumes:
...
 
-   ```conf
-   web:
-    volumes:
-      ./.docker/nginx/etc/nginx.conf:/etc/nginx/nginx.conf
-      ./.docker/nginx/etc/vhost.conf:/etc/nginx/conf.d/default.conf
+   ```yaml
+   services:
+     web:
+      volumes:
```
Comment (hguthrie) [inline src/cloud/docker/docker-containers-service.md]: looks like some indention problems in this yaml

Comment (shiftedreality) [reply src/cloud/docker/docker-containers-service.md]: Not sure, why it's different for yaml.

Fixed

**Comment by meker12** [comment]
running tests

**Pull request merged**